platform/src/main/java/org/stellar/anchor/platform/controller/CirclePaymentObserverController.java

```diff
@@ -0,0 +1,42 @@
+package org.stellar.anchor.platform.controller;
+
+import com.google.gson.Gson;
+import java.lang.reflect.Type;
+import java.util.Map;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import org.stellar.anchor.platform.paymentobserver.CirclePaymentObserverService;
+import shadow.com.google.common.reflect.TypeToken;
+
+@RestController
+@RequestMapping("/circle-observer")
+public class CirclePaymentObserverController {
```

Comment:
What is the purpose of this controller? Is this used for Circle to push notification to the `platform`?

Reply:
Yes, exactly! In the future, I believe we should move it to a separate service though. We can do it when we move the Stellar observer as well.